js/modules/k6/http/request.go

```diff
@@ -628,97 +638,36 @@ func (h *HTTP) request(ctx context.Context, preq *parsedHTTPRequest) (*HTTPRespo
 	return resp, nil
 }
 
+// Batch makes a multitude of requests in a batch. The provideds reqsV should be an array of request
```

Comment:
`Batch makes a multitude of requests in a batch.` doesn't sound quite right, and doesn't actually give much information. Something like `Batch makes multiple simultaneous HTTP requests` seems a bit better